l10n_ch_payment_slip/invoice.py

```diff
@@ -202,15 +199,19 @@ def action_number(self, cr, uid, ids, context=None):
                 cr, uid,
                 [('move_id', '=', inv.move_id.id),
                  ('account_id', '=', inv.account_id.id)],
+                order='id',
                 context=context
             )
             if not move_line_ids:
                 continue
             move_lines = move_line_obj.browse(cr, uid, move_line_ids,
                                               context=context)
+            sequence_int = 0
             for move_line in move_lines:
+                sequence_int += 1
                 if inv.type in ('out_invoice', 'out_refund'):
-                    ref = move_line.get_bvr_ref()
+                    ref = move_line.get_bvr_ref(sequence_int=sequence_int,
+                                                context=context)
```

Comment:
the context should never be passed manually when a method is called on a browse_record, it is already contained in the record when `browse()` is called.